ecs-cli/modules/cli/regcreds/regcreds_app_helpers.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2015-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"). You may
+// not use this file except in compliance with the License. A copy of the
+// License is located at
+//
+//	http://aws.amazon.com/apache2.0/
+//
+// or in the "license" file accompanying this file. This file is distributed
+// on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+// express or implied. See the License for the specific language governing
+// permissions and limitations under the License.
+
+package regcreds
+
+import (
+	"fmt"
+	"regexp"
+
+	"github.com/aws/amazon-ecs-cli/ecs-cli/modules/utils"
+	"github.com/aws/aws-sdk-go/aws"
+)
+
+func buildOutputEntry(arn string, key string, containers []string) CredsOutputEntry {
+	return CredsOutputEntry{
+		CredentialARN:  arn,
+		KMSKeyID:       key,
+		ContainerNames: containers,
+	}
+}
+
+// returns the provided value with the ecs-cli resource prefix added
+func generateECSResourceName(providedName string) *string {
+	return aws.String(utils.ECSCLIResourcePrefix + providedName)
+}
+
+func generateSecretDescription(regName string) *string {
+	return aws.String(fmt.Sprintf("Created with the ECS CLI for use with registry %s", regName))
+}
+
+func generateSecretString(username, password string) *string {
+	return aws.String(`{"username":"` + username + `","password":"` + password + `"}`)
+}
+
+func getExecutionRolePolicyARN(region string) string {
```

Comment:
nit: consider using something in the SDK for generating these ARNs, like https://docs.aws.amazon.com/sdk-for-go/api/aws/arn/